cli/ps.go

```diff
@@ -56,7 +66,63 @@ func (p *PsCommand) runPs(args []string) error {
 // psExample shows examples in ps command, and is used in auto-generated cli docs.
 func psExample() string {
 	return `$ pouch ps
-Name     ID       Status    Image                              Runtime
-1dad17   1dad17   stopped   docker.io/library/busybox:latest   runv
-505571   505571   stopped   docker.io/library/busybox:latest   runc`
+Name     ID       Status    Image                              Runtime   Created
+1dad17   1dad17   stopped   docker.io/library/busybox:latest   runv      about 1 hour ago
+505571   505571   stopped   docker.io/library/busybox:latest   runc      about 1 hour ago
+`
+}
+
+// createdTime is used to show the time from creation to now.
+func createdTime(created string) (s string) {
```

Comment:
please move this function to pkg, since it could be a common function which is used other places.
And add more unit-test for this function in pkg.